templates/firefox-ios.xliff

```diff
@@ -6317,7 +6321,7 @@ Sent from %2$@ 🦊 %3$@</source>
         <note>The label for the cancel button on the Terms of Service alert for the summarizer.</note>
       </trans-unit>
       <trans-unit id="Summarizer.ToS.Alert.FirefoxMessage.Label.v142" xml:space="preserve">
```

Comment:
Please always update string ID if the meaning of the string changes.
Documentation explains this [here](https://mozilla-l10n.github.io/documentation/products/mobile/mobile_l10n_faqs.html#when-do-i-have-to-use-a-different-id-when-updating-existing-strings).
CC @FilippoZazzeroni